requirements-doc.txt

```diff
@@ -42,3 +43,4 @@ types-Deprecated==1.2.9
 typing_extensions==4.3.0
 urllib3==1.26.12
 wrapt==1.14.1
+sphinx-autobuild
```

Comment:
how come `sphinx-autobuild` is listed twice?

Reply:
Mistake; fixed.